README.md

```diff
@@ -50,6 +50,35 @@ For example you might to create a set of pre-release npm packages for each commi
     filter: "@octopusdeploy" # Only include any packages with a scope of @octopusdeploy
 ```
 
+### `current-branch-name`
+
+This action allows you to get a sanitised branch details for use in subsequent GitHub action steps.
+
+#### Inputs
+
+| Name                | Description                                                                                                                            | Required |
+|---------------------|----------------------------------------------------------------------------------------------------------------------------------------|----------|
+| replacement_strings | This is the csv string of strings to replace in the branch name. Default value of "/"                                                  | Y        |
```

Comment:
I believe github actions has support for multi-input strings, which would allow them be to supplied as an array (either with syntax `[]` or as a multi-line string. I wonder if this would provide a bit easier usage experience as compared to csv?

Reply:
We might not expose this multi-input function with our wrapper in the shared utils but likely would be easy to add if we needed.

Reply:
I also wonder whether the name of the input here could be a bit more clear, `replacement_strings` kind of sounds like its the thing that will be used as the replacement, not sure if there is a better name? What do we do for the find-and-replace action (which this is effectively replacing)?

Reply:
Have renamed this to additional_strings_to_replace and made the action take a multiline input